### PR TITLE
disabled css for button

### PR DESCRIPTION
**Pull request by gags88**

### CODE REVIEW

File: src/kirby/components/button/button.component.tns.global.scss [hunk 1]
```diff
@@ -19,4 +19,11 @@ Button[kirby-button] {
     border-radius: 5;
   }
 
+  &[disabled],
```
Comment (jakobe) [inline src/kirby/components/button/button.component.tns.global.scss]: Do we need both?

Comment (gags88) [reply src/kirby/components/button/button.component.tns.global.scss]: Actually not.. removed

File: src/app/examples/button-example/button-example.component.html [hunk 2]
```diff
@@ -3,4 +3,5 @@
 <button kirby-button expand="block">Kirby Block Button</button>
 <button kirby-button expand="block" shape="square">Kirby Square Block Button</button>
 <button kirby-button colortype="primary">Primary Button</button>
-<button kirby-button colortype="secondary">Secondary Button</button>
\ No newline at end of file
+<button kirby-button colortype="secondary">Secondary Button</button>
+<button kirby-button shape="round" disabled>Disabled Button</button>
```
Comment (jakobe) [inline src/app/examples/button-example/button-example.component.html]: We don't use `shape` anymore
```suggestion
<button kirby-button disabled>Disabled Button</button>
```

File: src/app/examples/button-example/button-example.component.tns.html [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-  <Button kirby-button text="Kirby Block Button" expand="block"></Button>        
-  <Button kirby-button text="Round Block Cake Buttons" expand="block" shape="round"></Button>       
+  <Button kirby-button text="Kirby Rounded Button" shape="round"></Button>
+  <Button kirby-button text="Kirby Block Button" expand="block"></Button>
+  <Button kirby-button text="Round Block Cake Buttons" expand="block" shape="round"></Button>
   <Button kirby-button text="Primary Button" shape="round" colortype="primary"></Button>
   <Button kirby-button text="Secondary Button" shape="round" colortype="secondary"></Button>
-</StackLayout>
-
+  <Button kirby-button text="Disabled Button" shape="round" isEnabled="false"></Button>
```
Comment (jakobe) [inline src/app/examples/button-example/button-example.component.tns.html]: We don't use `shape` anymore
```suggestion
  <Button kirby-button text="Disabled Button" isEnabled="false"></Button>
```

Comment (gags88) [reply src/app/examples/button-example/button-example.component.tns.html]: Removed

File: src/app/examples/button-example/button-example.component.html (hunk 2 above)
Comment (gags88) [reply src/app/examples/button-example/button-example.component.html]: Removed

**Pull request merged**